*** a/src/Application.vala	2018-09-11 23:39:55.000000000 +0200
--- b/src/Application.vala	2018-09-24 00:10:03.169165591 +0200
*************** public class Sequeler.Application : Gtk.
*** 60,65 ****
--- 60,67 ----
  
      protected override void activate () {
          var window = new Sequeler.Window (this);
+         Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+         Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
          this.add_window (window);
      }
  }
